packages/forest-cloud/src/commands/bootstrap.ts

```diff
@@ -43,7 +43,21 @@ export default (program: Command, context: MakeCommands) => {
         logger.spinner.succeed('Environment found');
 
         const httpServer = buildHttpServer(vars);
-        await askToOverwriteCustomizationsOrAbortCommand(logger, httpServer);
+
+        const details = await httpServer.getLastPublishedCodeDetails();
+
+        if (details) {
+          displayCustomizationInfo(logger.spinner, details);
+          logger.spinner.warn(
+            // eslint-disable-next-line max-len
+            'If you continue it will generate a new boilerplate without your previous customizations',
```

Comment:
your previous customizations -> they are propably not his

```suggestion
            'If you continue it will boostrap a new customization project from scratch',
```